cypress/support/fragments/settings/users/manualCharges.js

```diff
@@ -61,6 +62,18 @@ export default {
     // eslint-disable-next-line cypress/no-unnecessary-waiting
     cy.wait(300);
   },
+  selectOwnerByName(owner) {
+    cy.do(ownerSelect.choose(owner));
+    cy.wait(1000);
+  },
+  checkSelectedOwner(owner) {
+    cy.wait(1000);
+    cy.get('#select-owner option').first().should('have.text', owner);
```

Comment:
Please, add 
    defaultOptionText:(el) => el.querySelector('select option:checked').textContent,
to interactor/select.js filters
And use it in the method
 cy.expect(
      ownerSelect.has({ defaultOptionText: owner })
    );
    instead of  using pure cypress